app/lib/account/models.dart

```diff
@@ -201,6 +201,9 @@ class SessionData {
   /// The time when the session will expire.
   final DateTime expires;
 
+  /// The time when the session was last authenticated.
+  final DateTime? authenticated;
```

Comment:
```suggestion
  final DateTime? authenticatedAt;
```

Reply:
oh, that is for the other PR, I'll remove it